src/v/cluster/types.h

```diff
@@ -279,7 +284,7 @@ struct init_tm_tx_request
     operator<<(std::ostream& o, const init_tm_tx_request& r);
 
     auto serde_fields() {
-        return std::tie(tx_id, transaction_timeout_ms, timeout);
+        return std::tie(tx_id, transaction_timeout_ms, expected_pid, timeout);
```

Comment:
this looks like it should break because the order of the std::tie is the order of the data on the wire. so v0 would decode the pid as the timeout. is there something i'm missing here?